/**:
  ros__parameters:

    # for marker detection algorithm
    resolution: 0.05
    # A sequence of low/high intensity to perform pattern matching.
    # -1: low intensity, 0: not consider, 1: high intensity
    intensity_pattern: [-1, -1, 0, 1, 1, 1, 1, 1, 0, -1, -1]
    match_intensity_difference_threshold: 20
    positive_match_num_threshold: 3
    negative_match_num_threshold: 3
    vote_threshold_for_detect_marker: 20

    # for interpolate algorithm
    self_pose_timeout_sec: 1.0
    self_pose_distance_tolerance_m: 1.0

    # for avoiding miss detection
    limit_distance_from_self_pose_to_marker_from_lanelet2: 5.0
    limit_distance_from_self_pose_to_marker: 2.0

    # base_covariance
    # [TBD] This value is dynamically scaled according to the distance at which markers are detected.
    base_covariance: [0.04, 0.0,  0.0,  0.0,        0.0,        0.0,
                      0.0,  0.04, 0.0,  0.0,        0.0,        0.0,
                      0.0,  0.0,  0.01, 0.0,        0.0,        0.0,
                      0.0,  0.0,  0.0,  0.00007569, 0.0,        0.0,
                      0.0,  0.0,  0.0,  0.0,        0.00007569, 0.0,
                      0.0,  0.0,  0.0,  0.0,        0.0,        0.00030625]
